k2/csrc/CMakeLists.txt

```diff
@@ -1,67 +1,51 @@
+# Make project root dir as the INCLUDE_DIRECTORIES of all tagets in csrc/.
+# The header should be included as `#include "k2/csrc/.."`, to avoid conflicts.
 include_directories(${CMAKE_SOURCE_DIR})
 
-# please sort the source files alphabetically
-add_library(fsa
-  arcsort.cc
-  aux_labels.cc
-  connect.cc
-  determinize.cc
-  fsa.cc
-  fsa_equivalent.cc
-  fsa_renderer.cc
-  fsa_util.cc
-  intersect.cc
-  properties.cc
-  rmepsilon.cc
-  topsort.cc
-  util.cc
-  weights.cc
+add_subdirectory(host)
```

Comment:
`old` may be a better name than `host`, as actually those files will be deleted finally. All host and device code now are in `*.cu` files.

Reply:
done